unit_tests/intersection/intersect_problem_00.msh

```diff
@@ -0,0 +1,159 @@
+$MeshFormat
```

Comment:
Wrong intersection extracted from bigger mesh. Seems it is still not covered by new testsuite.